components/bl-accordion-component/src/helpers.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+export function useItemsState(accordionData, onClick, controlledAccordion) {
+  const [stateStore, setStateStore] = useState(() => {
+    const stateStore = {};
+
+    if (!Array.isArray(accordionData)) {
```

Comment:
What if accordionData = number/undefined/any non array value?

Reply:
In this case, the component returns null from index.js:
```
if (!Array.isArray(accordionData)) {
  return stateStore;
}
```

In hook we use it to avoid blocking the page loading through the method forEach.

Reply:
I think it's a bad UX, we must validate that and log an error if validation failed.

Because if I understand correctly for now it works like this:

We pass invalid items
Components not rendered
We don't understand why